crates/extensions/tedge_flows/src/actor.rs

```diff
@@ -163,6 +171,38 @@ impl FlowsMapper {
         watch_requests
     }
 
+    async fn notify_flows_status(&mut self) -> Result<(), RuntimeError> {
+        let status = "enabled";
+        let now = OffsetDateTime::now_utc();
+        for flow in self.processor.flows.keys() {
+            let status = Self::flow_status(flow, status, &now);
+            self.mqtt_sender.send(status).await?;
+        }
+        Ok(())
+    }
+
+    async fn update_flow_status(&mut self, flow: &str) -> Result<(), RuntimeError> {
+        let now = OffsetDateTime::now_utc();
+        let status = if self.processor.flows.contains_key(flow) {
+            "updated"
+        } else {
+            "removed"
+        };
+        let status = Self::flow_status(flow, status, &now);
+        self.mqtt_sender.send(status).await?;
+        Ok(())
+    }
+
+    fn flow_status(flow: &str, status: &str, time: &OffsetDateTime) -> MqttMessage {
+        let topic = Topic::new_unchecked("te/device/main/service/tedge-flows/status/flows");
+        let payload = json!({
+            "flow": flow,
```

Comment:
I'm wondering if it'd be better to include the `flow` name in the **topic** instead of the payload so that these status messages can be published as **retained** messages per flow. That'll help the user get an overview of the status of all the flows by subscribing to `status/flow/<flow-name>` topic.